src/Overlay.js

```diff
@@ -1,13 +1,26 @@
 /*eslint-disable object-shorthand, react/prop-types */
```

Comment:
Why don't you just add propTypes for
`container`
`containerPadding`
`target`
`placement`
?

--
It is not a problem. I can take care of it after merging. Just added it to my todo-list.